sonic-thermalctld/scripts/thermalctld

```diff
@@ -463,13 +463,22 @@ class TemperatureUpdater(logger.Logger):
         state_db = daemon_base.db_connect("STATE_DB")
         self.table = swsscommon.Table(state_db, TemperatureUpdater.TEMPER_INFO_TABLE_NAME)
 
+        self.is_chassis_system = chassis.is_modular_chassis()
+        if self.is_chassis_system:
+            my_slot = chassis.get_my_slot()
+            table_name = TemperatureUpdater.TEMPER_INFO_TABLE_NAME+'_'+str(my_slot)
+            global_state_db = daemon_base.db_connect("GLOBAL_STATE_DB")
```

Comment:
Another comment is for a change needed in daemon_base (https://github.com/Azure/sonic-buildimage/blob/215ce13890b3cbffa56b097a3d9463cd3bc89447/src/sonic-py-common/sonic_py_common/daemon_base.py#L26). Since in this case the DB is not local we cannot use the UNIX socket, need to change the swsscommon.DBConnector() call to use the TCP connection.

Reply:
The TCP connection works when tested. The 3rd arg is for tcp-connection, correct?
 swsscommon.DBConnector(db_name, REDIS_TIMEOUT_MSECS, True, namespace)